dj_gui_api_server/DJConnector.py

```diff
@@ -165,7 +205,7 @@ def fetch_tuples(jwt_payload: dict, schema_name: str, table_name: str):
 
             # Add the row list to tuples
             rows.append(row)
-        return rows
+        return rows, len(query)
```

Comment:
Is the len return needed? I typically would just aspect users from of this api to use count the rows

Reply:
@Synicix Perhaps this is not clear and will serve a good opportunity to update our docs to make it clear. Would you propose where to update so that the following is clear?

With the introduction of paging, it now is no longer clear when the number of records (or the last page for that matter) ends. Each page would return the maximum amount per `limit` which if left as before, would require 'paging' through until you reach the first page less than the `limit`. Since this would be inefficient and it is standard practice to show user the maximum number of pages, I have added the `total_count` to be returned as well. This is a value that won't change between changing pages and represents all the records involved in the `restriction`. On the frontside, we could do `roundup(total_count/limit)` to determine the number of pages to pre-render for the user.

Reply:
Ah it is total records, I thoguth it was the number of records in rows. That makes a lot more sense now

Reply:
I think the docs you have are good, it just that I didn't really see the return in that jumbo mess.